<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <widget class="QPushButton" name="pushButtonChangePassword">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResponse">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>391</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>51</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Email Address</string>
      </property>
      <property name="buddy">
       <cstring>lineEditEmailAddress</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Old Password</string>
      </property>
      <property name="buddy">
       <cstring>lineEditOldPassword</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditEmailAddress"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditOldPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditNewPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditRePassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>New Password</string>
      </property>
      <property name="buddy">
       <cstring>lineEditOldPassword</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Re-enter New Password</string>
      </property>
      <property name="buddy">
       <cstring>lineEditOldPassword</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditEmailAddress</tabstop>
  <tabstop>lineEditOldPassword</tabstop>
  <tabstop>lineEditNewPassword</tabstop>
  <tabstop>lineEditRePassword</tabstop>
  <tabstop>pushButtonChangePassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonChangePassword</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>ChangePassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ChangePassword()</slot>
 </slots>
</ui>
